### ISSUE TITLE
Can't get the AAB (App Bundle) file.

### ISSUE THREAD

**Issue by parisetflorian** [FlutterFlow/flutterflow-issues#2022]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I followed this tutorial precisely:
https://docs.flutterflow.io/settings-and-integrations/app-settings/mobile-deployment/google-play-store-deployment

In point 4.1, the button to download the AAB never appears.
Instead I get a build error:
<img width="1171" alt="Screenshot 2023-12-18 at 11 08 23" src="https://github.com/FlutterFlow/flutterflow-issues/assets/7931903/bf1a938e-e2bb-4ab3-adf6-6616b272b147">
<img width="618" alt="Screenshot 2023-12-18 at 11 08 45" src="https://github.com/FlutterFlow/flutterflow-issues/assets/7931903/ada79fd0-5f34-4fb7-b3dd-46163f48e552">

### Expected Behavior

A button should appear to download the AAB file when building for the internal track. 

### Steps to Reproduce

Go to : https://app.flutterflow.io/project/eleven-labs-reader-zm52sy?tab=appSettings&appSettingsTab=mobileDeployment
Click "Deploy to the Google Play Store" in the Google Play Store section.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4Kk8n13812oblE1rr+cMB7/SkvJkwkU400lupEEUk0IZSrG7YMS8+kMxJVXemndGNEL06jrz4dwPSOvuzyA+UoITCbQaU4yqpAZznhTj2gR6bSCZaeREFADf5JJVPC0J+7mxNAHMRadHst2mCQf9ivHwLqJYP7OFgdJ+b8BIPVjE2qQ0SLb3kNh1JWeC/v

### Context

I'm trying to deploy on the Google Play Store.

### Visual documentation

<img width="1171" alt="Screenshot 2023-12-18 at 11 08 23" src="https://github.com/FlutterFlow/flutterflow-issues/assets/7931903/1d0358c2-0def-4371-91dd-a45262f4ce18">
<img width="618" alt="Screenshot 2023-12-18 at 11 08 45" src="https://github.com/FlutterFlow/flutterflow-issues/assets/7931903/693f1ee0-8251-48e1-a599-96a4a2433370">

### Environment

```markdown
- FlutterFlow version: 3.13.7
- Platform: Mac OS
- Browser name and version: Arc 1.20.1
- Operating system and version affected: Sonoma 14
```

**Comment by wenkaifan0720** [comment]
Hi @parisetflorian. If you look into the error message, you can see this: `Error: Dart library 'dart:html' is not available on this platform`. See this article (https://medium.com/@aashish.silswal/error-dart-library-dart-html-is-not-available-on-this-platform-import-dart-html-97dd8a15f2a) for more information. We generally don't help with custom code issues, so I will close this issue for now. But I think you have enough information to proceed.

**Comment by parisetflorian** [comment]
@wenkaifan0720 Thanks, that helps a lot. I missed that line :)
